mne/io/brainvision/tests/test_brainvision.py

```diff
@@ -77,82 +77,50 @@ def test_orig_units(recwarn):
     assert orig_units['ReRef'] == 'C'
 
 
-def test_vmrk_meas_date():
-    """Test successful extraction of measurement date."""
-    # Test file that does have a specific date
-    raw = read_raw_brainvision(vhdr_path)
-    assert_allclose(raw.info['meas_date'], [1384359243, 794231])
-    assert '2013-11-13 16:14:03 GMT' in repr(raw.info)
+@pytest.fixture()
+def _get_date_test_data(tmpdir):
```

Comment:
If I'm not mistaken we should be using the `tmpdir_factory` fixture, not the `tmpdir`. We should check the doc.